README.rst

```diff
@@ -0,0 +1,183 @@
+.. |DiscordLogo| image:: https://img.icons8.com/color/48/000000/discord-logo.png
+   :target: https://discordapp.com
+
+|DiscordLogo| OH-Bot
+======================================
+
+The Discord Office Hour Butler
+------------------------------
+
+.. raw:: html
+
+   <p align=center style="font-size:large">
+      <a href=#purpose>Purpose</a> • 
+      <a href=#overview>Overview</a> • 
+      <a href=#quickstart>Quick Start</a> • 
+      <a href=https://ecs-oh-bot.github.io/OH-Bot/docs/build/html/index.html>Docs</a> •
+      <a href=#license>License</a>
+   </p>
+.. raw:: html
+
+   </p>
+
+Purpose
+-------
+
+With the need for online learning becoming increasingly higher,
+efficient means of reaching educators has become extremely important. We
+implemented a discord bot to help instructors automate the process of
+their Office Hours through Discord.
+
+We created this bot with the goal of allowing students to effectively
+communicate with their instructors, with the option to be able to ask
+their peers for help while they wait.
+
+Overview
+--------
+
+**OH-Bot** is a server managment automation bot. This means that tasks
+like notifying and moving students when they are ready to been seen are
+handled by **OH-Bot**
+
+**OH-Bot** is a *self-hotsted* bot - meaning that you will need to host
+and maintain your own instance. See `Quick Start <#quickstart>`__ to
+get started.
+
+The standard OH-Session protocol that we follow: 
+
+#. Instructors Opens OH-Queue 
+#. Students enter queue and enter waiting room 
+#. Instructors dequeue students 
+#. Instructors close queue when OH-Session is over
```

Comment:
I think these bullet points are outdated. The architecture of BoB is that each instructor is associated with one or more queues, and that doesn't seem to be communicated here.